devtools/cli/src/main/java/io/quarkus/cli/RegistryAddCommand.java

```diff
@@ -39,6 +39,9 @@ public Integer call() throws Exception {
             config = registryClient.resolveConfig().mutable();
         }
         registryClient.refreshRegistryCache(output);
+        if (config.getRegistries() == null) {
+            output.warn("There is no registry set in the config file.");
```

Comment:
I would move this warning a few lines down.

AFTER adding registries, let's see if the count == 1, and the registry id is not equal to `io.quarkus.registry.Constants.DEFAULT_REGISTRY_ID`. If you look at the example add commands, they could add two registries out of the gate, which would avoid this warning.

So. IFF this is a new config (!existing) AND they have only one registry AND it is not the default registry, we should warn them that "whatever-id-they-used" is the only registry configured, and that it replaced the default.